1_fetch/src/sb_read_filter_by_comids.R

```diff
@@ -0,0 +1,43 @@
+#' Function to filter sb datasets to given comids in AOI. Generates unzip if needed.  
+
+sb_read_filter_by_comids <- function(comid, data_path, sb_comid_col = 'COMID',
+                                     selected_cols_contains = NULL, cbind = TRUE){
+  
+  #' @description function to unzip, fetch and munge datasets from Sciencebase. Built for STATSGO but generalized for ther SB datasets
+  #' @param comid character string or vector of character strings containing the common identifier (COMID) of the desired flowline(s)
+  #' @param data_path folder path to files downloaded from sb
+  #' @param sb_comid_col str of the the column name from the comids. Default is 'COMID' as this is typically the name of this column from ds from SB0
```

Comment:
What does `ds from SB0` refer to? Are those specific statsgo data files?

Reply:
not sure what i am refering to there 🤔, i think I mean `column in datasets from SB`. I'll update.